velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -237,6 +238,8 @@ Writer::Writer(
   }
   options_.timestampUnit =
       options.parquetWriteTimestampUnit.value_or(TimestampUnit::kNano);
+  options_.timestampTimeZone =
```

Comment:
"" is not a valid timestamp, do we have a default timestampTimeZone?

Reply:
If not, we should let it be std::nullopt, not empty timestampTimeZone

Reply:
@jinchengchenghh Updated.

Reply:
What's your basis for the default value? Can you add a comment here?

Reply:
@jinchengchenghh There is no basis. TimeZone based on the time zone where the program is running . Do you have a recommended value?

Reply:
In that case, you could make it std::nullopt

Reply:
@jinchengchenghh Set the default to std::nullopt.